tools/build_runtimes.sh

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+# This script will build the images for the given runtime and tag them using the
+# version numbers found in the ./versions directory.
+#   $1, the Docker repository to use, defaults to gcr.io/$PROJECT_ID
+
+# Exit on error or undefined variable
+set -eu
+
+readonly workspace=$(dirname $0)/..
+
+# If no repo is given get it from the ambient project.
+if [ -z "${1:-}" ]; then
+    readonly project_id=$(gcloud config list core/project --format="csv[no-heading](core)" | cut -f 2 -d '=')
```

Comment:
Optional:

`gcloud config list core/project --format="csv[no-heading](core)" | cut -f 2 -d '='`
 ->
 `gcloud config list --format="value(core.project)"`

Reply:
Thank you for this, simplified the scripts with this.f